metricbeat/module/kibana/stats/_meta/fields.yml

```diff
@@ -4,6 +4,7 @@
     Kibana stats and run-time metrics.
   release: ga
   fields:
+    # Service.id?
```

Comment:
👍

Reply:
Agreed. Will open separate PR to make this change.

Reply:
PR created: https://github.com/elastic/beats/pull/10350